Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2373,6 +2373,20 @@ DisallowSyncXHRDuringPageDismissalEnabled:
     WebCore:
       default: true
 
+DisallowSystemFontFallbackForPrivateUseAreaCharacters:
+  type: bool
+  status: stable
+  category: css
+  humanReadableName: "Disallow system font fallback for Private Use Area (PUA) characters"
+  humanReadableDescription: "Disallow font fallback using system fonts for charactes that are within the Private Use Area (PUA) block"
```

Comment:
Maybe mention that the default state here matches the specified behavior for PUA stuff.